Makefile

```diff
@@ -9,12 +13,23 @@ test:
 
 lint: lint-py lint-sh
 
+lint/docker: lint-py lint-sh/docker
+
 lint-py:
 	@tox -e lint
 
-lint-sh:
-	@shellcheck -s bash ubuntu-advantage modules/*
-	@shellcheck -s dash update-motd.d/*
+lint-sh: SHELLCHECK = shellcheck
+lint-sh: BASH_SCRIPTS = $(UA_SCRIPTS)
+lint-sh: DASH_SCRIPTS = $(MOTD_SCRIPTS)
+lint-sh: _lint-sh-command
+
```

Comment:
Is this grouping of makefile targets the same as having just one target and the command lines below it, indented with a tab? Or what is its purpose?

Reply:
Target-specific variable declarations need to be on a target line on their own.
You can declare the same target multiple times, they're all run in the order of declaration.
What you can't do is declare a variable as part of a target command, IOW you can't
```
lint-sh:
     BASH_SCRIPTS = $(UA_SCRIPTS)
     ....
     <commands>
```

Reply:
Nice trick